Amazon.QLDB.Driver/driver/AsyncQldbDriver.cs

```diff
@@ -0,0 +1,214 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.QLDB.Driver
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Amazon.Runtime;
+
+    /// <summary>
```

Comment:
Please strip all docstrings + implementation. They shouldnt be there to begin with, and it will make reading future PRs impossible and a pain in the butt.

Reply:
Removed docstrings for async methods/classes (but left them for methods I don't see changing).